### ISSUE TITLE
[Bug]: `eask create` does not honour user-specified versions

### ISSUE THREAD

**Issue by Phundrak** [emacs-eask/eask#40]
# Expected behaviour
When running `eask create`, the user gets asked which version the package should be at, as well as which version of Emacs should be the base dependency of the former. These choices should be reflected in the generated `.el` file.

# Actual behaviour

<details>
<summary>Running eask create</summary>

```
$ eask create erroneous-versions
Cloning into 'erroneous-versions'...
remote: Enumerating objects: 27, done.
remote: Counting objects: 100% (27/27), done.
remote: Compressing objects: 100% (18/18), done.
remote: Total 27 (delta 8), reused 21 (delta 5), pack-reused 0
Receiving objects: 100% (27/27), 4.22 KiB | 4.22 MiB/s, done.
Resolving deltas: 100% (8/8), done.
✓ Done cloning the template project

Initialize the Eask-file for your project...
package name: (erroneous-versions) 
version: (1.0.0) 0.1.0
description: Some package
entry-point: (erroneous-versions.el) 
emacs-version: (26.1) 24.1
About to write to /tmp/eask-test/erroneous-versions/Eask:

(package "erroneous-versions"
         "0.1.0"
         "Some package")

(package-file "erroneous-versions.el")

(source "gnu")

(depends-on "emacs" "24.1")


Is this OK? (yes) yes
Running Eask in the development environment
Press Ctrl+C to cancel.

Executing script inside Emacs...

✓ Checking Emacs version 29.0.50... done!
✓ Checking system gnu/linux... done!
Package-file seems to be missing ‘erroneous-versions.el’
✓ Loading Eask file in /tmp/eask-test/erroneous-versions/Eask... done!

Preparing your new elisp project... Package vc-mtn is deprecated
done ✓

Congratulations! Your new Elisp project is created in /tmp/eask-test/erroneous-versions/

  [1] Navigate to /tmp/eask-test/erroneous-versions/
  [2] Try out the command ‘eask info‘

Visit https://emacs-eask.github.io/ for quickstart guide and full documentation.
```

</details>

<details>
<summary>Eask file (user-defined versions honoured)</summary>

```emacs-lisp
(package "erroneous-versions"
         "0.1.0"
         "Some package")

(package-file "erroneous-versions.el")

(source "gnu")

(depends-on "emacs" "24.1")
```

</details>

<details>
<summary>Generated Emacs-Lisp file (user-defined versions not honoured)</summary>

```emacs-lisp
;;; erroneous-versions.el --- Some package  -*- lexical-binding: t; -*-

;; Copyright (C) 2022  Lucien Cartier-Tilet

;; Author: Lucien Cartier-Tilet <lucien@phundrak.com>
;; Maintainer: Lucien Cartier-Tilet <lucien@phundrak.com>
;; URL: https://github.com/user/repo
;; Version: 1.0.0
;; Package-Requires: ((emacs "26.1"))
;; Keywords:

;; This file is not part of GNU Emacs.

;; This program is free software: you can redistribute it and/or modify
;; it under the terms of the GNU General Public License as published by
;; the Free Software Foundation, either version 3 of the License, or
;; (at your option) any later version.

;; This program is distributed in the hope that it will be useful,
;; but WITHOUT ANY WARRANTY; without even the implied warranty of
;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
;; GNU General Public License for more details.

;; You should have received a copy of the GNU General Public License
;; along with this program. If not, see <https://www.gnu.org/licenses/>.

;;; Commentary:
;;
;; Some package
;;

;;; Code:

;; Happy coding! ;)

(provide 'erroneous-versions)
;;; erroneous-versions.el ends here
```

</details>

As you can see above, the `.el` file contains the following two lines, which do not honour the earlier user input.
```emacs-lisp
;; Version: 1.0.0
;; Package-Requires: ((emacs "26.1"))
```

`eask info` points the package version out, but misses the version mismatch between Eask’s Emacs version and the package’s.

<details>
<summary>Details</summary>

```
$ eask info
Running Eask in the development environment
Press Ctrl+C to cancel.

Executing script inside Emacs...

✓ Checking Emacs version 29.0.50... done!
✓ Checking system gnu/linux... done!
✓ Try constructing the package-descriptor (erroneous-versions.el)... succeeded! 
Unmatched version ’0.1.0’; it should be ’1.0.0’
✓ Loading Eask file in /tmp/eask-test/erroneous-versions/Eask... done!


erroneous-versions (0.1.0) | deps: 0 | devDeps: 0
Some package
https://github.com/user/repo

entry: erroneous-versions.el
kind: single

dist
.total-files: 1
.unpacked-size: 1.1k
```

</details>

# Steps to reproduce
1. Run `eask create` with a dummy project name
2. Enter the same input as I did in the first logs shared above

# Eask version
```
$ eask --version
0.6.31
```

**Comment by jcs090218** [comment]
Thanks for the detailed bug report! ❤️ 

I confirmed this is a bug! Should be fixed in #41. Can you try the latest version `0.6.32`?

Thanks!

**Comment by Phundrak** [comment]
It indeed works, thanks!

--
Lucien “Phundrak” Cartier-Tilet
<https://phundrak.com> (Français)
<https://phundrak.com/en> (English)
Sent from GNU/Emacs
